versioned_docs/version-3.0.x/concepts-messaging.md

```diff
@@ -1073,7 +1073,7 @@ Pulsar has two features, however, that enable you to override this default behav
 
 :::tip
 
-All message retention and expiry are managed at the [namespace](#namespaces) level. For a how-to, see the [Message retention and expiry](cookbooks-retention-expiry.md) cookbook.
+All message retention and expiry are managed at the [namespace](#namespaces) level or at topic level from Pulsar `2.6.0` activating `topicLevelPoliciesEnabled=true` at broker.conf. Since `2.11` that default value is `true`. For a how-to, see the [Message retention and expiry](cookbooks-retention-expiry.md) cookbook.
```

Comment:
We introduce topic-level policies from `2.6.0`, refer to https://github.com/apache/pulsar/pull/4955, but I think we support retention policies at the topic level from `2.7.0`, refer to https://github.com/apache/pulsar/pull/7747.

Make the default value of the configuration `topicLevelPoliciesEnabled` as `true` from `2.11.0`, refer to https://github.com/apache/pulsar/pull/15619.